scripts/itineraries/ItineraryList.js

```diff
@@ -29,6 +29,7 @@ const render = (arrayOfItineraries) => {
             <p>Park: ${itineraryObj.parkName}</p>
             <p>Eatery: ${itineraryObj.eateryName}</p>
             <p>Oddity: ${itineraryObj.oddityName}</p>
+            <button class="directionButton" id="directions--${itineraryObj}">Get Directions</button>
```

Comment:
This isn't going to work unfortunately, you can't really just put an entire object into the DOM like this via string interpolation. If you look at the click event handler, where you grab the second half of this attribute value, if you log it it is actually just the string `[Object object]`.

I think the way to approach this would be instead to:
1. Interpolate in the ID of the itinerary object on this `<button>` id.
1. In the event listener, grab that ID, grab the itinerary object with that ID from the itinerary data, then set that object you find into the `itineraryObj` property of the custom event detail.